airflow/gcp/operators/tasks.py

```diff
@@ -22,11 +22,17 @@
 which allow you to perform basic operations using
 Cloud Tasks queues/tasks.
 """
+from typing import Tuple, Sequence, Union, Dict, Optional
+
+from google.api_core.retry import Retry
+from google.cloud.tasks_v2.types import Queue, FieldMask, Task
 
 from airflow.gcp.hooks.tasks import CloudTasksHook
 from airflow.models import BaseOperator
 from airflow.utils.decorators import apply_defaults
 
+MetaData = Sequence[Tuple[str, str]]
```

Comment:
I like the idea. The metadata appears in many places. Maybe we can add this type to `GoogleBaseHook` file or in `airflow.typing`? @mik-laj @potiuk @TobKed WDYT?

Reply:
Sounds good for me. @mik-laj @potiuk WDYT?

Reply:
I am afraid that this will make reading much more difficult. This is not a very complicated type.  We will delete a few lines of code, but we will not lose the code, because imports will take much more characters.

Reply:
It would be cool if Google provided this type.

Reply:
after @mik-laj explanations I've changed my mind.

Reply:
Yeah. agree with @mik-laj  Let's optimise for reading not writing. Reading 'Sequence[Tuple[str, str]]' tells you much more than 'MetaData'.

Reply:
Changes applied with new fixup.